runtimes/nodejs/actions/initialHtml.js

```diff
@@ -15,9 +15,13 @@ async function main(args) {
   const putCORSAction = `/${namespace}/cloud-object-storage/bucket-cors-put`;
   const fileName = 'userProfileImg';
   const blocking = true;
-  // const options = { ignore_certs: true };
-  const ow = openwhisk();
+  const ignore_certs = args.ignore_certs ? args.ignore_certs : false
+  if(ignore_certs){
+    process.env['__OW_IGNORE_CERTS'] = true
```

Comment:
you dont need to do this to initialize `npm openwhisk` to ignore the certification validation, you can pass the flag `ignore_certs` directly to the module.